### ISSUE TITLE
how it start? 

### ISSUE THREAD

**Issue by AIN108** [eiaserinnys/pdf2md#1]
I tried, but I don't know how to use the PDF file I have, and there are only two sites I can use. Also, when I try to use it after connecting to Microsoft Edge and Chrome with the PDF connector, I get the following directory error(RuntimeError: Directory 'static/' does not exist). I couldn't solve it with pip install frontend etc.

**Comment by eiaserinnys** [comment]
I apologize for the inconvenience. 

If you have requirement.txt installed, you can run it in two ways

1. python -m src.main --f (filename or URL)
2. python -m src.main (with the URL of the pdf file copied to the clipboard)

The 'static/' folder issue can be caused by installing pip install fitz, please install pip install PyMuPDF after pip uninstall fitz. Please check to see if the issue persists after doing so.
https://stackoverflow.com/questions/67982760/raise-runtimeerrorfdirectory-directory-does-not-exist-runtimeerror-dire

**Comment by AIN108** [comment]
I still have a question, when I access the URL, pdf with the institution, license, etc. python -m src.main --f (ex jstor, acm, nature, etc.) I get mostly errors (no module).(This varies from time to time) Is it possible to use only the huggingspace and archive x in the example?

**Comment by eiaserinnys** [comment]
Yes, you are correct. As of now, special handling is only implemented for arxiv and huggingface (you can check the `try_download` in `main.py` for more details). When I have some spare time, I'll look into adding support for other sites you've mentioned!
